examples/carplay-web-app/src/worker/carplay.ts

```diff
@@ -13,10 +11,14 @@ let carplayWeb: CarplayWeb | null = null
 let config: Partial<DongleConfig> | null = null
 
 const handleMessage = (message: CarplayMessage) => {
-  if (message instanceof VideoData) {
```

Comment:
Discovered a bug here - these messages are not video or audio data so it always fell back to the last postMesssage.

Without utilizing transferable Objects this deep clones all video and audio data. fixed in this PR, just need to resolve the type issue.